cmd/runhcs/container.go

```diff
@@ -234,6 +235,7 @@ func (c *container) startVMShim(logFile string, opts *uvm.UVMOptions) (*os.Proce
 
 type containerConfig struct {
 	ID                     string
+	Owner                  string
```

Comment:
Are these ever marshalled/unmarshalled? Wondering if it needs an omitempty?

Reply:
`continerConfig` is not but `persistedState` is saved to the registry. I think its fine to store it in all cases to make debugging easier since its not going across the wire but I can follow up with a "add json opts to `persistedState`" issue if you want?

Reply:
Sure, SGTM.